addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -173,6 +173,9 @@ class Applicant(models.Model):
     meeting_ids = fields.One2many('calendar.event', 'applicant_id', 'Meetings')
     meeting_display_text = fields.Char(compute='_compute_meeting_display')
     meeting_display_date = fields.Date(compute='_compute_meeting_display')
+    # UTMs - enforcing the fact that we want to 'set null' when relation is unlinked
+    medium_id = fields.Many2one(ondelete='set null')
```

Comment:
Why not the campaign_id ? As the "job" one is managed in the campaign override, other campaigns can be set to null, isn't it ?

Reply:
It was not on the form view but I guess it makes sense anyway so I added it.